omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/ObsResource1_8.java

```diff
@@ -526,8 +526,7 @@ public Object upload(MultipartFile file, RequestContext context) throws Response
 		
 		obs = obsService.saveObs(obs, null);
 		
-		SimpleObject ret = (SimpleObject) ConversionUtil.convertToRepresentation(obs, Representation.DEFAULT);
-		return ret;
+		return ConversionUtil.convertToRepresentation(obs, Representation.DEFAULT);
```

Comment:
Does this line return SimpleObject   objtect type without casting?

Reply:
Have updated the code base

Reply:
Have updated the code base

> Does this line return SimpleObject objtect type without casting?

Have updated the code base

Reply:
cc: @sacull